swift-blackjack-viewsTests/PlayerSpec.swift

```diff
@@ -0,0 +1,283 @@
+//  PlayerSpec.swift
+
+import Quick
+import Nimble
+@testable import swift_blackjack_views
+
+class PlayerSpec: QuickSpec {
+    override func spec() {
+        describe("Player") {
+            
+            var player = Player(name: "Player")
+            
+            let aceOfSpades : Card = Card(suit: "♠︎", rank: "A")
+            let aceOfHearts : Card = Card(suit: "♥︎", rank: "A")
+            let queenOfHearts : Card = Card(suit: "♥︎", rank: "Q")
+            let tenOfDiamonds : Card = Card(suit: "♦︎", rank: "10")
+            let twoOfClubs : Card = Card(suit: "♣︎", rank: "2")
+
+            beforeEach({ () -> () in
```

Comment:
Yah, so if a function/method takes a closure as its final argument, you can provide all the remaining arguments in parens and then do the closure in brackets after, like `wooWee(1) { ... }` is equal to `wooWee(1, { ... })`. And the `() -> () in ...` is the explicit argument-and-return-type-for-the-closure syntax, but in simple cases it can figure it out without anything. Put 'em together and what have you got? `beforeEach { ... }`